types/validtransaction.go

```diff
@@ -60,9 +60,13 @@ func (t Transaction) correctFileContracts(currentHeight BlockHeight) error {
 		}
 		outputPortion := PostTax(currentHeight, fc.Payout)
 		if validProofOutputSum.Cmp(outputPortion) != 0 {
+			println("valid proof problems")
 			return ErrFileContractOutputSumViolation
 		}
 		if missedProofOutputSum.Cmp(outputPortion) != 0 {
+			println("missed proof problems")
+			println(missedProofOutputSum.String())
+			println(outputPortion.String())
```

Comment:
aw man, I've tried to use `git diff` to prevent that from happening. I worry that they are in more places too though. This is definitely some of the sloppiest code I've written in a while.